### DISCUSSION TITLE
Should the License of a debugsource package be inherrited from SourceLicense tag?

### DISCUSSION THREAD

**Discussion by hroncok** [rpm-software-management/rpm#3035] [Q&A]
I wonder if the License of the generated debugsource package should be derived from the SourceLicense tag or not. Do we assume the debugsource package contains sources, hence it is covered by SourceLicense, or do we assume it only contains sources of stuff that has been built, and hence should not inherit SourceLicense?

**Comment by pmatilai** [comment]
I guess you'd need a lawyer to answer this one, but the autogenerated debugsource can only contain sources of stuff that was built.
